frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -222,35 +222,36 @@ const fetchNodeMetrics = (): Promise<NodeMetrics> => {
   const metrics = [
     {
       key: 'usedMemory',
-      query: 'sum by (instance) (node_memory_MemTotal_bytes - node_memory_MemAvailable_bytes)',
+      query: 'node_memory_MemTotal_bytes - node_memory_MemAvailable_bytes',
     },
     {
       key: 'totalMemory',
-      query: 'sum by (instance) (node_memory_MemTotal_bytes)',
+      query: 'node_memory_MemTotal_bytes',
     },
     {
       key: 'usedStorage',
-      query: 'sum by (instance) (node_filesystem_size_bytes - node_filesystem_free_bytes)',
+      query: 'instance:node_filesystem_usage:sum',
```

Comment:
This recording rule is not the same as previous query. Source: https://github.com/openshift/cluster-monitoring-operator/blob/master/assets/prometheus-k8s/rules.yaml#L714-L716